themes/default/layouts/partials/head.html

```diff
@@ -144,9 +144,6 @@
         </script>
     {{ end }}
 
-    <!-- Preload the component JavaScript to help our Google Page Speed score. -->
-    <link rel="preload" href="/js/components/components.esm.js" as="script" crossorigin="anonymous">
```

Comment:
I don't see this in the `assets` partial in https://github.com/pulumi/theme/pull/23. Is this also an accidental deletion?

Reply:
Note that this is a preload, so maybe this is not needed anymore with the new structure?

Reply:
No this is intentional — we no longer load Stencil separately this way; it gets bundled into theme/assets/js/bundle.js.